cluster_support_bot/hydra.py

```diff
@@ -1,4 +1,4 @@
-import errors
+from . import errors
```

Comment:
[PEP 8][1] would probably prefer:

```python
import datetime

import requests

from . import errors
```

But I'm fine as this stands too.

[1]: https://www.python.org/dev/peps/pep-0008/#imports